core/src/main/java/org/apache/iceberg/SchemaUpdate.java

```diff
@@ -136,13 +136,15 @@ private void internalAddColumn(String parent, String name, boolean isOptional, T
           parentField.type().isNestedType() && parentField.type().asNestedType().isStructType(),
           "Cannot add to non-struct column: %s: %s", parent, parentField.type());
       parentId = parentField.fieldId();
+      Types.NestedField currField = schema.findField(parent + "." + name);
```

Comment:
Is "current" too long? For style, we prefer using full words rather than abbreviations in most cases.

Reply:
ack.